### PR TITLE
Bug fixed.

### PR DESCRIPTION
**Pull request by AxelBelin**
Fixed: A call to endpoint POST researches execute return a 500 internal error due to NPE during response DTO mapping.

### CODE REVIEW

**Review by PereiraE — APPROVED** [review]

File: src/main/java/fr/uge/lp/databackbonepoc/services/ResearchService.java [hunk 1]
```diff
@@ -83,10 +90,7 @@ public Research validateAndGet(long researchId) {
```
Comment (P-DAO) [inline src/main/java/fr/uge/lp/databackbonepoc/services/ResearchService.java:90]: javadoc ?

File: src/main/java/fr/uge/lp/databackbonepoc/models/local/Research.java [hunk 2]
```diff
@@ -73,4 +73,18 @@ public Research(
     this.leftEntityName = leftEntityName;
     this.rightEntityName = rightEntityName;
   }
+
```
Comment (P-DAO) [inline src/main/java/fr/uge/lp/databackbonepoc/models/local/Research.java:76]: javadoc

File: src/main/java/fr/uge/lp/databackbonepoc/controllers/mappers/ResearchMapper.java [hunk 3]
```diff
@@ -40,4 +41,21 @@ public ResearchResponse toResponse(Research research) {
         linkMapper.toResponse(research.getLink())
     );
   }
+
```
Comment (P-DAO) [inline src/main/java/fr/uge/lp/databackbonepoc/controllers/mappers/ResearchMapper.java:44]: javadoc

**Review by P-DAO — COMMENTED** [review]
manque de la javadoc

**Pull request merged**